src/components/TransactionPanel/TransactionPanel.js

```diff
@@ -318,6 +325,12 @@ export class TransactionPanelComponent extends Component {
       id: 'TransactionPanel.sendingMessageNotAllowed',
     });
 
+    const paymentMethodsPage = (
```

Comment:
Maybe renaming this to `paymentMethodsPageLink` would be more descriptive and make it easier to understand where this is used.